logging/tests/unit/handlers/test_handlers.py

```diff
@@ -45,7 +45,7 @@ def test_emit(self):
                                    None, None)
         handler.emit(record)
 
-        self.assertEqual(handler.transport.send_called_with, (record, message, _GLOBAL_RESOURCE))
+        self.assertEqual(handler.transport.send_called_with, (record, message, _GLOBAL_RESOURCE, None))
```

Comment:
This already had line length 97, sorry I missed it before. Now has length 103, so even worse. Instead:

```python
        self.assertEqual(
            handler.transport.send_called_with,
            (record, message, _GLOBAL_RESOURCE, None))
```